tools/utils/daymet_to_ats_box.py

```diff
@@ -112,8 +113,9 @@ def initData(d, vars, num_days, nx, ny):
 
 def collectDaymet(tmpdir, bounds, start, end, vars=None, force=False):
     """Calls the DayMet Rest API to get data and save raw data."""
-    if vars is None:
-        vars = VALID_VARIABLES
+
+#    if vars is None:
```

Comment:
Please keep this option.  This would let us download only one variable  (e.g. only air temp), and since each variable comes in it's own file and needs a separate download, I would like to have to have this option.  For instance, we may only need precip, and not the energy quantities, if we aren't doing a full radiation balance.